src/lib/components/TestResult.svelte

```diff
@@ -25,40 +26,41 @@
 	<h6 class="text-accent-foreground mb-2 text-[10px] font-semibold uppercase">
 		{testDetails.name}
 	</h6>
-	<h3 class="mb-1 text-lg font-semibold">Submitted Successfully</h3>
+	<h3 class="mb-1 text-lg font-semibold">{$t('submitted_successfully')}</h3>
```

Comment:
We should maintain the string,
```suggestion
	<h3 class="mb-1 text-lg font-semibold">{$t('Submitted Successfully')}</h3>
```